modules/tinymce/src/core/main/ts/api/dom/ControlSelection.ts

```diff
@@ -19,7 +20,6 @@ import { EditorEvent } from '../util/EventDispatcher';
 import Tools from '../util/Tools';
 import VK from '../util/VK';
 import EditorSelection from './Selection';
-
```

Comment:
This line shouldn't have been removed.

Reply:
Right eslint was giving me problems I think I moved the line but missed that it also removed the newline. Fixed now.